### PR TITLE
Adapt clients_last_seen for submission_date name change

### PR DESCRIPTION
**Pull request by jklukas**
Fixes [this morning's clients_last_seen failure](https://workflow.telemetry.mozilla.org/log?task_id=clients_last_seen&dag_id=main_summary&execution_date=2019-07-22T01%3A00%3A00%2B00%3A00)

### CODE REVIEW

**Comment by fbertsch** [comment]
I believed we need two additional changes:
- Remove `key_value` from experiments
- Resolve the type difference on `sample_id`

For the latter, we could:
- Cast it here as a string
- Move the historical data to a new location, and re-create `clients_daily_v6` as a view + new data on top of it, with the sample_id as an int

**Comment by jklukas** [comment]
Added a commit to document that the query is running in telemetry_raw. The clients_daily in the raw location has integer sample_id and does not have key_value removed, which should address both of your concerns.

**Comment by fbertsch** [comment]
Okay, just to clarify:
`telemetry_raw` has `clients_last_seen_raw_v1` and `clients_daily_v6`. We still run into an incompatible `sample_id`, because `clients_last_seen_raw_v1.sample_id` is a `string`, while `clients_daily_v6.sample_id` is an integer.

Did we not finish the schema changes? For clarity, please confirm what I said above - but it seems to me this is wrong: https://github.com/mozilla/bigquery-etl/blob/master/tests/clients_last_seen_raw_v1/test_single_day/clients_last_seen_raw_v1.schema.json#L53.

**Comment by jklukas** [comment]
You're right. We need to hold off on this for @relud's input. He may already have a plan in place for this situation.

**Comment by relud** [comment]
for now we should cast sample_id to a string in clients_last_seen_raw_v1.sql

File: templates/clients_last_seen_raw_v1.sql [hunk 1]
```diff
@@ -25,11 +27,11 @@ WITH
       1 AS bits
     FROM
       UNNEST(experiments.key_value)) AS days_seen_in_experiment,
-    * EXCEPT (submission_date_s3)
+    * EXCEPT (submission_date)
```
Comment (relud) [inline templates/clients_last_seen_raw_v1.sql]:
```suggestion
    * EXCEPT (submission_date) REPLACE (CAST(sample_id AS STRING) AS sample_id)
```

File: sql/clients_last_seen_raw_v1.sql [hunk 2]
```diff
@@ -87,11 +89,11 @@ WITH
       1 AS bits
     FROM
       UNNEST(experiments.key_value)) AS days_seen_in_experiment,
-    * EXCEPT (submission_date_s3)
+    * EXCEPT (submission_date)
```
Comment (relud) [inline sql/clients_last_seen_raw_v1.sql]: include the change here too for CI
```suggestion
    * EXCEPT (submission_date) REPLACE (CAST(sample_id AS STRING) AS sample_id)
```

**Pull request merged**